notebooks/notebooks/paper/estimate-flops-katago-v2.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    plt.legend(bbox_to_anchor=(1, 1), loc="upper left");
</pre><p>add axis labels to plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Added.